domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/accessprofiles/RoleToAccessProfileService.java

```diff
@@ -1,14 +1,18 @@
 package uk.gov.hmcts.ccd.definition.store.domain.service.accessprofiles;
 
-import java.util.List;
 import uk.gov.hmcts.ccd.definition.store.repository.entity.RoleToAccessProfilesEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.model.RoleAssignment;
 import uk.gov.hmcts.ccd.definition.store.repository.model.RoleToAccessProfiles;
 
+import java.util.List;
+
 public interface RoleToAccessProfileService {
 
     void saveAll(List<RoleToAccessProfilesEntity> entityList);
 
     List<RoleToAccessProfiles> findByRoleName(String roleName);
 
     List<RoleToAccessProfiles> findByCaseTypeReferences(List<String> caseTypeReferences);
+
+    List<RoleAssignment> findByCaseTypeId(String id);
```

Comment:
```suggestion
    List<RoleAssignment> findRoleAssignmentsByCaseTypeId(String id);
```
Since this method doesn't return a list of `RoleToAccessProfiles`

Reply:
si si , :) DONE